src/style.css

```diff
@@ -0,0 +1,150 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+a,
+img,
+p {
+  margin: 0;
+  padding: 0;
+  text-decoration: none;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  color: black;
+  height: 406px;
+  width: 198px;
```

Comment:
check the usage of [box-sizing: border-box;](https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing)